libobs/obs-nix-platform.h

```diff
@@ -24,7 +24,6 @@ extern "C" {
 #endif
 
 enum obs_nix_platform_type {
-	OBS_NIX_PLATFORM_X11_GLX,
```

Comment:
Removing this item, we loose binary compatibility with plugins. These code use the enumerator.
https://github.com/exeldro/obs-source-dock/blob/134f1f614a4382e68de265230aaddc06cc66c8b9/qt-display.cpp#L163-L178
https://github.com/norihiro/obs-color-monitor/blob/e8359b1560c1ea8dede019e8e59943325b48be28/src/obsgui-helper.hpp#L21-L36

Reply:
This is an excellent point that I didn't consider, thanks! Though now that I'm thinking, OBS 28 will break ABI anyway (Qt5 → Qt6) so I think we're accidentally safe for now.

Reply:
IIRC, since the package for Ubuntu 20.04 (PPA) will be still compiled with Qt5, isn't that still ABI compatible?

Reply:
Qt is not part of the ABI, but even if it were we should seek to reduce the pain not use it to add more. We can just add a _UNUSED to the name to signify its no longer used but just deleting it makes it that much harder to detect the ABI break as   a plugin developer.

Reply:
The enum value is still there now, but nothing can set it, and there's a comment instructing to stop using it in code. Is that good now?